src/Provider/Youtube/Provider.php

```diff
@@ -147,7 +149,17 @@ public function provide($input)
         }
 
         $response = $this->getHttpClient()->send($request, $options);
-
+		
+		//But just incase it doesn't, we can fallback to the old URL.
+		if (strpos($response->getBodyAsString(), 'status=fail') !== false) {
```

Comment:
`Response::getBodyAsString()` was removed in v0.8. Use `getBody()->__toString()` instead.

```diff
-        if (strpos($response->getBodyAsString(), 'status=fail') !== false) {
+        if (strpos($response->getBody()->__toString(), 'status=fail') !== false) {
```